capella_model_explorer/__main__.py

```diff
@@ -233,6 +234,11 @@ def main() -> None:
     show_default=True,
     help="The Docker image to use with '--container'.",
 )
+@click.option(
+    "--skip-build/--no-skip-build",
+    default=False,
+    help="Skip the initial asset rebuild",
```

Comment:
When the CSS is missing it will be built although `--skip-build` is passed. That shall be documented here.

Reply:
It already says "***re***build" in the help, isn't this enough?

Maybe we should also rename the flag to `--skip-rebuild` to match.